operators/pkg/tenant-controller/tenant_controller_test.go

```diff
@@ -211,11 +211,12 @@ var _ = Describe("Tenant controller", func() {
 			},
 		}
 		Expect(k8sClient.Create(ctx, tn)).Should(Succeed())
+		tnNm := &v1.Namespace{}
+		doesEventuallyExists(ctx, types.NamespacedName{Name: nsName}, tnNm, BeTrue(), timeout, interval)
```

Comment:
This is already checked in `checkTnClusterResourceCreation`.

Reply:
Removed